src/insertion-checker.ts

```diff
@@ -8,15 +8,7 @@ const warn =
   };
 
 export function trackInsertion(_lyra: unknown) {
-  const lyra = _lyra as object & { [kInsertions]?: number };
-
-  if (typeof lyra[kInsertions] !== "number") {
-    queueMicrotask(() => {
-      lyra[kInsertions] = undefined;
-    });
-
-    lyra[kInsertions] = 0;
-  }
```

Comment:
The `kInsertions` is always defined in the instance so it is no longer necessary to do so.